.github/workflows/deployExpensifyHelp.yml

```diff
@@ -46,24 +48,26 @@ jobs:
       - name: Deploy to Cloudflare Pages
         uses: cloudflare/pages-action@f0a1cd58cd66095dee69bfa18fa5efd1dde93bca
         id: deploy
-        if: github.event_name != 'pull_request' || (github.event_name == 'pull_request' && !github.event.pull_request.head.repo.fork)
+        if: env.IS_PR_FROM_FORK != 'true'
         with:
           apiToken: ${{ secrets.CLOUDFLARE_PAGES_TOKEN }}
           accountId: ${{ secrets.CLOUDFLARE_ACCOUNT_ID }}
           projectName: helpdot
           directory: ./docs/_site
 
       - name: Setup Cloudflare CLI
+        if: env.IS_PR_FROM_FORK != 'true'
         run: pip3 install cloudflare==2.19.0
 
       - name: Purge Cloudflare cache
+        if: env.IS_PR_FROM_FORK != 'true'
         run: /home/runner/.local/bin/cli4 --verbose --delete hosts=["help.expensify.com"] /zones/:9ee042e6cfc7fd45e74aa7d2f78d617b/purge_cache
         env:
           CF_API_KEY: ${{ secrets.CLOUDFLARE_TOKEN }}
 
       - name: Leave a comment on the PR
         uses: actions-cool/maintain-one-comment@de04bd2a3750d86b324829a3ff34d47e48e16f4b
-        if: ${{ github.event_name == 'pull_request' }}
+        if: ${{ github.event_name == 'pull_request' && env.IS_PR_FROM_FORK != 'true' }}
```

Comment:
> github.event_name == 'pull_request

NAB: This condition is already encompassed by `IS_PR_FROM_FORK`. It's not strictly necessary, but for clarity, we can also keep it.